requirements/deploy.txt

```diff
@@ -1,3 +1,8 @@
-gunicorn==19.9.0 \
-    --hash=sha256:aa8e0b40b4157b36a5df5e599f45c9c76d6af43845ba3b3b0efe2c70473c2471 \
-    --hash=sha256:fa2662097c66f920f53f70621c6c58ca4a3c4d3434205e608e121b5b3b71f4f3
+gunicorn==20.0.4 \
+    --hash=sha256:1904bb2b8a43658807108d59c3f3d56c2b6121a701161de0ddf9ad140073c626 \
+    --hash=sha256:cd4a810dd51bf497552cf3f863b575dabd73d6ad6a91075b65936b151cbf4f9c
+
+# The following packages are considered to be unsafe in a requirements file:
```

Comment:
shouldn't we have gotten the correct setup tools from the main requirements?

Reply:
Our CI dependency checker fails because `setuptools` is now a dependency for `gunicorn` and it treats each file individually. That's why #6985 was failing.

Reply:
👍